test/views/TIRCarnetReferenceViewSpec.scala

```diff
@@ -40,11 +39,11 @@ class TIRCarnetReferenceViewSpec extends InputTextViewBehaviours[String] {
 
   behave like pageWithBackLink
 
-  behave like pageWithCaption(messages("items.preHeading"))
+  behave like pageWithCaption("Items")
 
   behave like pageWithHeading()
 
-  behave like pageWithInputText(Some(InputSize.ThreeQuarterWidth))
+  behave like pageWithInputText()
 
   behave like pageWithSubmitButton("Save and continue")
```

Comment:
Should this not also be "Continue" ?

Reply:
This is the old p4 one so just going to leave as Save and Continue. It sounds like this will be getting deleted in the coming days anyway